Core/Font/PGF.cpp

```diff
@@ -616,7 +621,13 @@ void PGF::DrawCharacter(const GlyphImage *image, int clipX, int clipY, int clipW
 				value = consumeBits(4, fontData, bitPtr);
 			}
 
-			decodedPixels[pixelIndex++] = value | (value << 4);
+			// CONSERVATIVE: Keep original 8-bit expansion to ensure font visibility
```

Comment:
I guess it's possible that this might fix something, if indeed this is what happens on hardware. Does reverting this part change anything?